.rat-excludes

```diff
@@ -35,6 +35,7 @@ dagre-d3.min.js
 graphlib-dot.min.js
 sorttable.js
 vis.min.js
+vis.js
```

Comment:
I think it's because we can't expect the cluster to have network access to the external world so we need to bundle it.

But @sarutak what do we use `vis.map` for? The timeline seems to work fine even without `vis.js`. What are the specific UI differences?